app/javascript/flavours/polyam/styles/components/notification.scss

```diff
@@ -310,12 +310,20 @@
     }
   }
 
+  // Polyam: Style for icon buttons
+  &--follow &__actions {
+    @include follow-unfollow-icons;
+  }
```

Comment:
I don't really like that the style for the icon buttons has to be set this way.
Ideally, the style would be applied by using a common class in FollowIconButton, but that requires wrapping it.